src/TimeGrid.jsx

```diff
@@ -77,9 +80,23 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
+    }
+
     //this.checkOverflow()
   }
 
+  componentWillReceiveProps(nextProps) {
+    // When paginating, reset scroll
+    if (+nextProps.start !== +this.props.start) {
```

Comment:
I don't think you need all that complexity, if a dev wants to programmatically scroll the component then I think we want to just let that happen, it's up to them to not make that annoying for their users. if you want the current behavior then leave the prop static and no problems. 

my. main concern is that "on mount" is sometimes not a good time to initialization stuff with regard to layout. for instance if the calendar is hidden when mounted you can't do any scrolling.